python/calico/felix/profilerules.py

```diff
@@ -149,6 +156,11 @@ def on_profile_update(self, profile, force_reprogram=False):
         assert not self._dead, "Shouldn't receive updates after we're dead."
         self._pending_profile = profile
         self._dirty |= force_reprogram
+        if profile:
+            if self._untracked is None:
+                self._untracked = profile['untracked']
+            else:
+                assert profile['untracked'] == self._untracked
```

Comment:
Does this mean that we'll blow up if someone changes a policy from tracked to untracked (or is that handled elsewhere)?  That feels pretty mainline to me.

You could handle that in the manager by simulating a delete than a re-add.  We do that in the endpoint manager to support some awkward mutations.